### ISSUE TITLE
Timestamps for friends and follows relationships

### ISSUE THREAD

**Issue by igorbrigadir** [DocNow/twarc#590]
There may be a way of decoding the pagination tokens to extract an exact or approximate time when someone followed an account:

See https://twittercommunity.com/t/get-the-first-followers-of-an-account/165728/3?u=igorbrigadir and https://twitter.com/richie3366/status/1300420302141378560 

This would be extremely useful.

**Comment by edsu** [comment]
Wow! Maybe a good idea for a plugin?

**Comment by igorbrigadir** [comment]
Yeah, i haven't figured it out yet - and the person who got it to work somehow hasn't responded yet, but i'm pretty sure there's a timestamp encoded in the pagination token, and you could keep paginating a second at a time or something like that to timestamp follows - that's what i think is happening so far.

**Comment by edsu** [comment]
If you do hear more it would be interesting to also know how they discovered it!
